### PR TITLE
Add shipping methods for valid address

### PR DESCRIPTION
**Pull request by dkubb**
This branch changes the `spree/admin/orders#new` action so that shipping methods are added to the order if the address is known.

### CODE REVIEW

**Comment by dkubb** [comment]
@mbj this is ready for review. This has been verified with:

```bash
RAILS_ENV=test bundle exec mutant -r ./spec/dummy/config/environment.rb \
  --use rspec \
  -j1 \
  -- \
  'Spree::Admin::OrdersController#new' \
  'Spree::Order#empty!'
```

**Comment by dkubb** [comment]
@mbj this is ready for review. It has been validated with:

```bash
# backend
RAILS_ENV=test bundle exec mutant -r ./spec/dummy/config/environment.rb   --use rspec \
  -j1 \
  -- \
  'Spree::Admin::OrdersController#new'

# core
RAILS_ENV=test bundle exec mutant -r ./spec/dummy/config/environment.rb   --use rspec \
  -j1 \
  -- \
  'Spree::Order#empty!
```

File: backend/spec/controllers/spree/admin/orders_controller_spec.rb [hunk 1]
```diff
@@ -145,19 +159,21 @@ def do_request
...
-            create(:order_with_line_items)
-          end
-
           before { do_request }
 
           it 'uses the incomplete order' do
             expect(assigns[:order]).to eql(incomplete_order)
           end
+
+          it 'sets the order state to the expected state' do
```
Comment (mbj) [inline backend/spec/controllers/spree/admin/orders_controller_spec.rb]: Could any other state apart `'payment'` be valid at this point?

If so we should name the context `'sets the order to the most advanced state it is valid in'` or such.

Comment (mbj) [inline backend/spec/controllers/spree/admin/orders_controller_spec.rb]: Could any other state apart 'payment' be valid at this point?

If so we should name the context 'sets the order to the most advanced state it is valid in' or such.

Comment (dkubb) [inline backend/spec/controllers/spree/admin/orders_controller_spec.rb]: Good idea. I will fix this spec name.

**Comment by mbj** [comment]
@dkubb Code wise I'm fine apart from the minor flags. Will merge and validate on our project once green after the flags are addressed.

**Comment by dkubb** [comment]
@mbj I fixed.replied to all your flags. This should be ready once the build is green.

**Pull request merged**